src/main/kotlin/io/Ust.kt

```diff
@@ -192,17 +219,46 @@ object Ust {
             line.tryGetValue("VBR")?.let {
                 pendingVBR = it.split(',').mapNotNull { cell -> cell.toDoubleOrNull() }
             }
+            line.tryGetValue("Piches")?.let {
+                if (pendingNotePitches != null)
+                    return@let
+                pendingNotePitches = parseMode1PitchData(it)
+            }
+            line.tryGetValue("Pitches")?.let {
+                if (pendingNotePitches != null)
+                    return@let
+                pendingNotePitches = parseMode1PitchData(it)
+            }
+            line.tryGetValue("PitchBend")?.let {
+                if (pendingNotePitches != null)
+                    return@let
+                pendingNotePitches = parseMode1PitchData(it)
+            }
+        }
+        val pitchDataMode1 = UtauMode1TrackPitchData(notePitchDataListMode1)
```

Comment:
Convert to `null` if it's empty? like the next line does.

Reply:
Got it!

Reply:
Changed in 1eaef32 .